src/components/OperationsList/index.js

```diff
@@ -214,77 +221,108 @@ const mapDispatchToProps = {
     if (this.props.account !== nextProps.account) {
       return true
     }
 
-    if (this.props.withAccount !== nextProps.withAccount) {
+    if (this.props.accounts !== nextProps.accounts) {
       return true
     }
 
-    if (this.props.canShowMore !== nextProps.canShowMore) {
+    if (this.props.withAccount !== nextProps.withAccount) {
       return true
     }
 
-    if (this._hashCache === null) {
+    if (this.props.canShowMore !== nextProps.canShowMore) {
       return true
     }
 
-    return !isEqual(this._hashCache, this.getHashCache(nextProps.operations))
+    return false
   }
 
-  getHashCache = (operations: OperationType[]) => operations.map(t => t.id)
-
   handleClickOperation = (data: Object) => this.props.openModal(MODAL_OPERATION_DETAILS, data)
 
-  _hashCache = null
-
   render() {
-    const { account, canShowMore, onAccountClick, operations, t, title, withAccount } = this.props
+    const {
+      account,
+      title,
+      accounts,
+      canShowMore,
+      onAccountClick,
+      t,
+      withAccount,
+      nbToShow,
+    } = this.props
+
+    if (!account && !accounts) {
+      console.warn('Preventing render OperationsList because not received account or accounts') // eslint-disable-line no-console
+      return null
+    }
+    const groupedOperations = accounts
+      ? groupAccountsOperationsByDay(accounts, nbToShow || 20)
+      : groupAccountOperationsByDay(account, nbToShow)
 
-    this._hashCache = this.getHashCache(operations)
+    const accountsMap = accounts ? keyBy(accounts, 'id') : { [account.id]: account }
 
     return (
       <Defer>
-        <Box>
-          <Card flow={1} title={title} p={0}>
-            <Box>
-              {operations.map(op => {
-                // $FlowFixMe
-                const acc = account || op.account
-                return (
-                  <Operation
-                    account={acc}
-                    key={`${op.id}${acc ? `-${acc.id}` : ''}`}
-                    minConfirmations={acc.minConfirmations}
-                    onAccountClick={onAccountClick}
-                    onOperationClick={this.handleClickOperation}
-                    t={t}
-                    op={op}
-                    withAccount={withAccount}
-                  />
-                )
-              })}
-            </Box>
-          </Card>
+        <Box flow={4}>
+          {title && (
+            <Text color="dark" ff="Museo Sans" fontSize={6}>
+              {title}
+            </Text>
+          )}
+          {groupedOperations.map(group => {
+            const d = moment(group.day)
+            const isToday = d.isSame(today, 'day')
+            const isYesterday = d.isSame(moment(today).subtract(1, 'day'), 'day')
+            return (
+              <Box flow={2} key={group.day.toISOString()}>
+                <Box ff="Open Sans|SemiBold" fontSize={4} color="grey">
+                  {isToday ? 'Today' : isYesterday ? 'Yesterday' : d.format('D MMMM YYYY')}
```

Comment:
if you want an already internationalized version, you could do something like this: https://github.com/gre/ledger-wallet-mobile/blob/master/src/screens/Dashboard/SectionHeader.js 
(BTW see how we have chosen to display things like `April 10, 2018 – Today` , maybe we could suggest to design , we have room to display this , even on mobile ^^ )